src/App.js

```diff
@@ -13,7 +13,7 @@ const { persistor, store } = createStore();
 const App = () => (
   <Provider store={store}>
     <PersistGate
-      loading={<Splash />}
+      loading={null}
```

Comment:
Remove the Splash component from import because it is not used anymore in this file.

Run `yarn lint` and `yarn flow` before commit.